erts/doc/src/erlang.xml

```diff
@@ -1592,15 +1667,20 @@ b</pre>
 
     <func>
       <name name="error" arity="1" since=""/>
-      <fsummary>Stop execution with a specified reason.</fsummary>
-      <desc>
-        <p>Stops the execution of the calling process with the reason
-          <c><anno>Reason</anno></c>, where <c><anno>Reason</anno></c>
-          is any term. The exit reason is
-          <c>{<anno>Reason</anno>, Where}</c>, where <c>Where</c>
-          is a list of the functions most recently called (the current
-          function first). As evaluating this function causes
-          the process to terminate, it has no return value. Example:</p>
+      <fsummary>Raise exception of class error.</fsummary>
+      <desc>
+        <p>Raises exception of class <c>error</c> with the reason
+        <c><anno>Reason</anno></c>, where <c><anno>Reason</anno></c>
+        is any term. The exit reason is <c>{<anno>Reason</anno>,
```

Comment:
It is not an "exit reason". It should be either "error reason" or just "reason".

Reply:
Fixed: 3b5bfbb